mullvad-management-interface/src/client.rs

```diff
@@ -50,6 +51,7 @@ pub enum DaemonEvent {
     Device(DeviceEvent),
     RemoveDevice(RemoveDeviceEvent),
     NewAccessMethod(AccessMethodSetting),
+    FeatureIndicators(FeatureIndicators),
```

Comment:
I discussed with @hulthe before his vacation about whether this should be its own entry in `DaemonEvent`, or a part of the `TunnelState`. With this implementation, won't we very frequenty get double `DaemonEvent`s notifying the same thing? what are your thoughts?

Reply:
Also, since the indicators are not really meaningful in the `Disconnected` state, we could simply leave them out of that enum variant, which would reduce some ambiguity.

Reply:
I like it! `mullvad_types::TunnelState` is a representation of the actual tunnel state, so it should be fine to put non-critical pieces of user-facing information there.

Reply:
Dropped `FeatureIndicators` from the `DaemonEvent` enum, think it turned out better!